### DISCUSSION TITLE
Site design

### DISCUSSION THREAD

**Discussion by omgmog** [remotehack/remotehack.github.io#130] [Ideas]
_This is a discussion that started [on Slack](https://remote-hack.slack.com/archives/C022W679C3Y/p1622025261003000) and has been brought over to Github discussions to make it more transparent and easier to follow_

---

> Max:
> Just to discuss the design of the site. Some considerations that have come up:
> - adding external links to the nav (e.g. Twitter, Slack, TinyLetter)
> - reducing the size of the hero banner on the home, so that important stuff is above the fold
> - Remove the /podcast page, make it a redirect to /live, and improve the design of /live
> I'm gonna whip up some sort of design example, and then build it on a new branch

I've put the beginnings of this up on [a branch _design-tidy-up_](https://github.com/remotehack/remotehack.github.io/tree/design-tidy-up)

**Comment by omgmog** [comment]
![image](https://user-images.githubusercontent.com/50949/119980144-c92aab00-bfb3-11eb-914a-e13a280c4d3d.png)

> Max:
> If we wanted to push for a logo that depended less on that recovery kit photo, which of these would people prefer (or something different?) font is https://gt-cinetype.com/

**Reply by benfoxall** [reply]
For reference, this was the first design concept for the logo/site:

![image (1)](https://user-images.githubusercontent.com/51385/119982177-70104680-bfb6-11eb-9e75-5151ef08f610.png)

it had a bit of a printed/manual feel to it (which I didn't quite capture in the implementation).

**Comment by omgmog** [comment]
Related: #106, #105
